src/MagnumPlugins/AssimpImporter/Test/AssimpImporterTest.cpp

```diff
@@ -482,9 +482,9 @@ void AssimpImporterTest::animation() {
 
     constexpr Vector3 rotationData[KeyCount]{
         {0.0f, 0.0f, 0.0f},
-        {0.0f, 0.0f, Float{Rad(-90.0_degf)}},
-        {Float{Rad(90.0_degf)}, 0.0f, Float{Rad(-90.0_degf)}},
-        {Float{Rad(135.0_degf)}, 0.0f, Float{Rad(-90.0_degf)}}
```

Comment:
Any idea why GCC 4.8 doesn't like the curly braces here?

> error: cannot convert 'Magnum::Rad {aka Magnum::Math::Rad<float>}' to 'Magnum::Float {aka float}' in initialization
         {Float{Rad(135.0_degf)}, 0.0f, Float{Rad(-90.0_degf)}}

Reply:
As far as I know, early C++11 spec didn't allow using `{}` for conversion operators, only for constructors. Old Clang (3.8) has the same issue. Similar issue is with initializing references, e.g. `int& a{b}`doesn't compile there. Newer compilers are more consistent and allow `{}` for all those cases. 

C++17 then allows using `{}` for enum initialization as well (the main reason was to be able to do e.g. `std::byte{56}`).